protocol.md

```diff
@@ -298,31 +294,24 @@ The Client MAY supply the `Upload-Metadata` header to add additional metadata to
 upload creation request. The Server MAY decide to ignore or use this information
 to further process the request or to reject it.
 
-If the size of the upload exceeds the maximum which MAY be indicated using the
-`Tus-Max-Size` headers, the Server MUST respond with the
-`413 Request Entity Too Large` status code. 
-
-If an upload contains additional metadata responses to `HEAD` requests against
-these uploads MUST include the `Upload-Metadata` header and its value as sent in the
-upload creation request.
```

Comment:
good catch. I wanted to rephrase it. I forgot about that.
How about:
All `HEAD` responses to an upload URL MUST include the `Upload-Metadata` header and the value MUST be that of the one received during its creation.

or

If `Upload-Metadata` was set in creation request, the Server MUST include the header 
in all subsequent `HEAD` responses.